Automattic/jetpack
# Jetpack plugin: update Backup, Anti-spam, and VideoPress logos

This PR updates the VaultPress Backup, Akismet Anti-spam, and VideoPress logos in the Jetpack plugin.

Fixes 1202858161751496-as-1203427126857118

#### Changes proposed in this Pull Request:

`js-packages/components` 
- Updated the VaultPress Backup, Akismet Anti-spam, and VideoPress icons
- Added the possibility to set the icon fill color

`plugins/jetpack`
- Updated the VaultPress Backup, Akismet Anti-spam, and VideoPress
- Updated product names for VaultPress Backup, and Akismet Anti-spam
- Deleted unused icons

#### Other information:


#### Jetpack product discussion
Context: p1HpG7-jkl-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

- Spin up a test site
- Build `plugins/jetpack` and `js-packages/components`
- Install the Jetpack plugin
- Visit the Jetpack Dashboard and notice the logos and names are updated

<img width="1019" alt="Screen Shot 2022-12-08 at 4 58 45 PM" src="https://user-images.githubusercontent.com/1620183/206575893-b1e1c990-0ee8-495a-a7fc-ae9ffb1bceb5.png">


- Visit the _Stats_ page and notice the logos and names are updated

<img width="800" alt="Screen Shot 2022-12-08 at 4 56 46 PM" src="https://user-images.githubusercontent.com/1620183/206575579-16b3d218-252b-4aec-a69c-0402489d2f92.png">


- Visit the _My Jetpack_ page and notice the logos and names are updated

<img width="800" alt="Screen Shot 2022-12-08 at 4 50 54 PM" src="https://user-images.githubusercontent.com/1620183/206575038-f411612d-e5af-4db4-bc4d-7acc47aaa522.png">


- Get a VaultPress Backup, Akismet Anti-spam, and VideoPress subscriptions
- Visit the _My Plan_ page and notice the logos and names are updated

<img width="1004" alt="Screen Shot 2022-12-08 at 5 04 50 PM" src="https://user-images.githubusercontent.com/1620183/206576817-21c9f770-d99d-4a28-8ea2-ad98a642566c.png">
